packages/components/molecules/f-rating/src/assets/scss/rating.scss

```diff
@@ -35,10 +35,7 @@
   width: 32px;
 }
 
-.c-rating-star--filled path {
-    fill: f.$color-support-brand-01;
-}
-
+.c-rating-star--filled path,
 .c-rating-star--empty path {
-    fill: f.$color-mozzarella-50;
+    fill: f.$color-support-brand-01;
```

Comment:
Is there a shared class that this rule could live in?

Reply:
Do you mean use a class within the vue component and then define here?

Reply:
Should be fixed now.